src/RepositoryProxy.php

```diff
@@ -61,6 +61,16 @@ public function count(): int
         return \count($this->findAll());
     }
 
+    public function countWithCriteria(array $criteria): int
```

Comment:
at first, I wanted to add this `$criteria` parameter to `count()` but that's not possible because this class implements `\Countable`

Reply:
I would have thought this possible with a default value for the parameter, no?

Reply:
hum, you're right! I'll fix this

Reply:
done